packages/zoltan2/src/algorithms/partition/Zoltan2_AlgMultiJagged.hpp

```diff
@@ -1575,14 +1569,12 @@ void AlgMJ<mj_scalar_t, mj_lno_t, mj_gno_t, mj_part_t, mj_node_t>::
   this->mj_uniform_weights(0) = true;
 
   this->mj_weights = Kokkos::View<mj_scalar_t**, device_t>
-    ("weights", 1);
+    ("weights", 1, 1);
```

Comment:
1. I'm pretty sure that building with deprecated code off would have caught this; please do so in the future.  Thanks!
2. Will these objects always be 1 x 1?

Reply:
Yes, not building with deprecated off was the start of my troubles and these errors were caught by deprecated off tests. Will do in the future. The weights will not always be 1x1.

Reply:
It's alright -- Trilinos really should have PR tests with deprecated code both on and off.  Not sure if that's the case yet.  Do you need those weights to be allocated initially?  If not, you could just leave them default constructed.